### ISSUE TITLE
(SameSpot+Achtung) Way to Bulk Order Colonists to Same Spot?

### ISSUE THREAD

**Issue by maarxx** [pardeike/SameSpot#1]
I am using both SameSpot and Achtung. I love them both! Excellent mods!

I am wondering if it is possible to select a group of pawns and instruct them all to walk onto the same spot, stacked?

If it is already a feature, I seem to have missed it.

Currently all I can figure out to do is select the colonists one-by-one and order them into the same spot.

If there is not a feature to bulk order to same spot, would you please consider adding it?

I am not sure if it would be a feature of SameSpot or a feature of Achtung, but I posted it here.

Perhaps a feature of SameSpot, when a keyboard key is held down simultaneously as a modifier? Just a thought.

**Comment by pardeike** [comment]
Did you try to hold shift?

Samespot contains this code:
```csharp
static bool DestinationIsReserved(this PawnDestinationManager instance, IntVec3 c, Pawn searcher)
{
	if (Find.Selector.SelectedObjects.Count() == 1) return false;
	if (Input.GetKey(KeyCode.LeftShift) || Input.GetKey(KeyCode.RightShift)) return false;
	return instance.DestinationIsReserved(c, searcher);
}
```

**Comment by maarxx** [comment]
Huzzah! I disabled Achtung for first test and confirmed yes, SHIFT provides the behavior I was looking for! Excellent!

It is not documented anywhere? I checked again the README and the Steam page and didn't see it.

When I enabled Achtung again, the feature was overriden by Achtung's "Move colonists relative", which was also set to SHIFT, which is why I had trouble discovering the feature. (( I think this is default? I don't recall changing it. But maybe I did. I was using Achtung for a while already before using SameSpot. ))

I changed Achtung's "Move colonists relative" to CTRL and now all the features seem to be available.

Most excellent! Thanks again for the great mods!
